eks-without-vpc/providers.tf

```diff
@@ -0,0 +1,26 @@
+#
+# Provider Configuration
+#
+
+provider "aws" {
+  version = "~> 1.60.0"
+  region = "${var.provider-region}"
```

Comment:
`profile` can be deleted (also the var), since this information is automatically taken from aws cli configuration

Reply:
deleted